pom.xml

```diff
@@ -28,7 +28,7 @@
     <maven.surefire.version>2.22.2</maven.surefire.version>
     <karate.junit.version>1.2.0</karate.junit.version>
     <graal.version>21.3.4</graal.version>  <!-- TODO: remove when karate-junit5 no longer ships with vulnerable graal, see below -->
-    <junit.version>5.8.2</junit.version>
+    <junit.version>5.7.2</junit.version>
```

Comment:
what is the reason downgrade the JUnit version?

Reply:
For some reason, mod-search tests are failing with the following error if the version is 5.8.2. Downgrading it to 5.7.2 makes it possible to run tests successfully. 
![image](https://user-images.githubusercontent.com/85455914/198019629-fae45be5-956d-4364-b739-724bdeaf57d1.png)

Reply:
It happens because of @ParameterizedTest. 
We will remove them in ticket [FAT-3340](https://issues.folio.org/browse/FAT-3340)